### ISSUE TITLE
no bond for modified nucleotide

### ISSUE THREAD

**Issue by hainm** [arose/ngl#47]
Please check this

```python
import pytraj as pt
import nglview as nv

traj = pt.load('f2.pdb')
view = nv.show_pytraj(traj)
view.representations = []
view.add_representation('licorice')
view
```

![picture1](https://cloud.githubusercontent.com/assets/4451957/12610995/99ef39f6-c4b8-11e5-9e6a-446f83347120.png)

file: https://github.com/hainm/amber_things/blob/master/things/nucleic/f2.pdb

(modified nucleotides: AF2, GF2).

Does ngl has function to guess bond? if not, does it have option to add bond_indices (pytraj has bond info).

**Comment by arose** [comment]
Unfortunately the modified nucleotides are not detected as a part of the polymer chain. I will keep that in mind when introducing `residueTypes` which will allow some more costlier calculations to detect such things.

Currently you can add bonds only via CONNECT records in the PDB file. However I am working on a version that allows efficient adding of bonds from other sources.

**Comment by hainm** [comment]
use CONNECT is perfect solution for pytraj. Check my PR

https://github.com/arose/nglview/pull/25

**Comment by arose** [comment]
Cool. I leave this open however as I intend to have better automatic bonding.

**Comment by arose** [comment]
It is now better in the dev branch. However with your example file the lower part is recognized as DNA due to the missing `O2'` atoms and thus not connected to the upper RNA part. Complicated...

![screenshot 59](https://cloud.githubusercontent.com/assets/272250/14401010/200716de-fdb8-11e5-9219-11b15afa89b2.png)

**Comment by arose** [comment]
I think I figured it out, the O2' atoms are replaced by F2' atoms. Handled in 251b481acfd2e8ef9e06652852cb58ffe4532cc6. Do you know/have examples of other common modifications like that?

![screenshot 24](https://cloud.githubusercontent.com/assets/272250/18017078/94f11438-6b84-11e6-9213-baf3b18158e1.png)

**Comment by hainm** [comment]
I will check and post later.

**Comment by hainm** [comment]
PDB ID 
- [ ] 1XV6

**Comment by arose** [comment]
close this as the initial problem is fixed, track remaining problems in #192
